.ci/jenkins_job_build.sh

```diff
@@ -31,11 +47,47 @@ fi
 # Signify to all scripts that they are running in a CI environment
 [ -z "${KATA_DEV_MODE}" ] && export CI=true
 
+if [ $# -eq 0 ]; then
+	usage >&2
+	exit 1
+fi
+
 # Name of the repo that we are going to test
 export kata_repo="$1"
+if [[ "${kata_repo}" != *"github.com/kata-containers/"* ]]; then
+	echo "Invalid repository name" >&2
+	usage >&2
+	exit 1
+fi
 
 echo "Setup env for kata repository: $kata_repo"
 
+shift
+test_type="all"
+
+while [[ $# -ne 0 ]]; do
+	option=$1
+	case ${option} in
+	-t|--type)
+		if [[ $# -lt 2 ]]; then
+			echo "Missing parameter after ${option}" >&2
+			exit 1
+		fi
+		shift
+		parameter=$1
+		case ${parameter} in
```

Comment:
Nit: Since https://github.com/kata-containers/tests/pull/4926/files#diff-2b38dc854ae065a444de231fcf9436e6954ee0b9c71052edda190c1c0a72f5e6R290-R302 is checking the `test_type`, it's safe to assign `$parameter` to `test_type` directly.

Reply:
Thanks for the comment. I will reflect this on the code. :wink:

Reply:
Ah, but in this case, this check could function as an early catch for a wrong parameter, which does not make the script go through `set.up` until the error is identified. What do you think?